nixpkgs_review/github.py

```diff
@@ -1,23 +1,35 @@
+import sys
 import json
+import os
+import urllib.error
 import urllib.parse
 import urllib.request
```

Comment:
```suggestion
from urllib import error, parse, request
```

Reply:
Unfortuately that doesn't work, since they're used in the code as, for example, `urllib.error` rather than as `error`.

Reply:
Wouldn't it be enough to just import urllib then?

Reply:
No — it doesn’t automatically import the sub modules. I tested that.